### PR TITLE
Hammer task list parser for 1.x branch

### PR DESCRIPTION
**Pull request by PaulWay**
The output of the `hammer task list` command is useful for diagnosing problems with Satellite running tasks.

Full documentation and test coverage included.

### CODE REVIEW

File: insights/parsers/tests/test_hammer_task_list.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+from ...parsers import hammer_task_list
+from ...tests import context_wrap
+
+import doctest
+
+
+def test_HTL_doc_examples():
+    failed, tested = doctest.testmod(hammer_task_list)
+    assert failed == 0
```
Comment (bfahr) [inline insights/parsers/tests/test_hammer_task_list.py]: @PaulWay I'm assuming that you are using your Examples as your positive tests?

**Comment by bfahr** [comment]
@PaulWay are you missing the change to specs.py to add the new parser spec?

File: insights/parsers/tests/test_hammer_task_list.py [hunk 2]
```diff
@@ -3,9 +3,37 @@
...
+07780e8f-dd81-49c4-a792-c4d4d162eb10,,foreman_admin,2016/11/11 06:55:50,2016/11/11 06:55:51,stopped,error,Update for host sat62disc.example.org,"There was an issue with the backend service candlepin: 404 Resource Not Found, There was an issue with the backend service candlepin: 404 Resource Not Found"
+749a17a1-a8cb-46f0-98f6-017576481df8,,foreman_admin,2016/11/11 06:51:28,2016/11/11 06:51:29,stopped,error,Update for host sat62disc.example.org,"There was an issue with the backend service candlepin: 404 Resource Not Found, There was an issue with the backend service candlepin: 404 Resource Not Found"
+d8f41819-b492-46e5-b0e3-ead3b4b6810c,,foreman_admin,2016/11/11 06:51:22,2016/11/11 06:51:28,stopped,error,Package Profile Update,500 Internal Server Error
+'''
+
 
 def test_HTL_doc_examples():
-    failed, tested = doctest.testmod(hammer_task_list)
+    from insights.parsers.hammer_task_list import HammerTaskList
```
Comment (xiangce) [inline insights/parsers/tests/test_hammer_task_list.py]: Would you mind to move this `import` to the top of the file?

File: insights/parsers/tests/test_hammer_task_list.py (hunk 1 above)
Comment (PaulWay) [reply insights/parsers/tests/test_hammer_task_list.py]: Yep, definitely!  No need to write two tests for the one thing.

**Comment by PaulWay** [comment]
@bfahr ROFL - completely forgot the spec!  Fixed!

**Comment by xiangce** [comment]
@PaulWay - I think this PR is OK to be merged with the resubmission to the master branch. Please resubmit it when you are free.

**Pull request merged**